index.js

```diff
@@ -14,7 +16,35 @@ app.get('/', async (req, res) => {
 });
 
 // TODO change to post when finish testing
-app.get('/create-pdf', async (req, res) => {
+app.get('/certificate', async (req, res) => {
+  if (!req.query.id) {
+    //TODO ERROR HANDLING
+    res.status(400).send('Missing id');
+    return;
+  }
+  //Getting the buyer's name from the query if it exists
+  const nameRu = req.query.nameRu;
+  const nameHe = req.query.nameHe;
+  const chapterDoc = await Chapter.findOne({ id: +req.query.id });
+  if (nameRu || nameHe) {
```

Comment:
nested ifs are hard to read.

Please rewrite to this logic
```
maybeUpdateChapter(chapter, nameRu, nameHe) // if nameRu and nameHe exists and buyer not set - set, if buyer set - throw an error
getBuyer() // throw error if no buyer
renderPdf(buyer)
```

Reply:
Use `guard clause` inside maybeUpdateChapter as per https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
Even better instead of maybeUpdateChapter - add method maybeSetBuyer to chapter model